packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkEdit.vue

```diff
@@ -1,11 +1,7 @@
 <template>
   <div class="oc-files-edit-public-link oc-files-file-link-form" data-testid="new-files-link">
     <form @submit.prevent>
-      <transition
-        enter-active-class="uk-animation-slide-top-small"
-        leave-active-class="uk-animation-slide-top-small uk-animation-reverse"
-        name="custom-classes-transition"
-      >
+      <transition name="custom-classes-transition">
```

Comment:
The list transitions are from older times. You can drop the transition tag entirely for now. We'll have to come up with something nicer.